MainModule/Server/Core/Logs.lua

```diff
@@ -282,62 +283,67 @@ return function(Vargs)
-				end
-				if pcall(function() loadstring("local hi = 'test'") end) then
-					det.Loadstring='Enabled'
-				else
-					det.Loadstring='Disabled'
-				end
-				if service.Workspace.FilteringEnabled then
-					det.Filtering="Enabled"
-				else
-					det.Filtering="Disabled"
-				end
-				if service.Workspace.StreamingEnabled then
-					det.Streaming="Enabled"
-				else
-					det.Streaming="Disabled"
-				end
-				det.NilPlayers = nilplayers
-				det.PlaceName = service.MarketPlace:GetProductInfo(game.PlaceId).Name
-				det.PlaceOwner = service.MarketPlace:GetProductInfo(game.PlaceId).Creator.Name
-				det.ServerSpeed = service.Round(service.Workspace:GetRealPhysicsFPS())
-				--det.AdminVersion = version
-				det.ServerStartTime = service.GetTime(server.ServerStartTime)
 				local nonnumber=0
 				for i,v in pairs(service.NetworkServer:children()) do
 					if v and v:GetPlayer() and not Admin.CheckAdmin(v:GetPlayer(),false) then
 						nonnumber=nonnumber+1
 					end
 				end
-				det.NonAdmins=nonnumber
 				local adminnumber=0
 				for i,v in pairs(service.NetworkServer:children()) do
 					if v and v:GetPlayer() and Admin.CheckAdmin(v:GetPlayer(),false) then
 						adminnumber=adminnumber+1
 					end
 				end
-				det.CurrentTime=service.GetTime()
-				det.ServerAge=service.GetTime(os.time()-server.ServerStartTime)
-				det.Admins=adminnumber
-				det.Objects=#Variables.Objects
-				det.Cameras=#Variables.Cameras
-				
-				local tab = {}
-				for i,v in pairs(det) do
-					table.insert(tab,{Text = i..": "..tostring(v),Desc = tostring(v)})
+				table.insert(tab,{Text = "―――――――――――――――――――――――"})
+				table.insert(tab,{Text = "Place Name: "..service.MarketPlace:GetProductInfo(game.PlaceId).Name})
+				table.insert(tab,{Text = "Place Owner: "..service.MarketPlace:GetProductInfo(game.PlaceId).Creator.Name})
+				table.insert(tab,{Text = "―――――――――――――――――――――――"}) 
+				table.insert(tab,{Text = "Server Speed: "..service.Round(service.Workspace:GetRealPhysicsFPS())})
+				table.insert(tab,{Text = "Server Start Time: "..service.GetTime(server.ServerStartTime)})
+				table.insert(tab,{Text = "Server Age: "..service.GetTime(os.time()-server.ServerStartTime)})
+				table.insert(tab,{Text = "―――――――――――――――――――――――"})
+				if HTTP.CheckHttp() then
+					table.insert(tab,{Text = "HTTPService: [ON]"})
+				else
+					table.insert(tab,{Text = "HTTPService: [OFF]"})
+				end
+				if game.Workspace.AllowThirdPartySales == true then
+					table.insert(tab,{Text = "Third Party Sales: [ON]"})
+				else
+					table.insert(tab,{Text = "Third Party Sales: [OFF]"})
+				end
+
+				if pcall(function() loadstring("local hi = 'test'") end) then
+					table.insert(tab,{Text = "Loadstring: [ON]"})
+
+				else
+					table.insert(tab,{Text = "Loadstring: [OFF]"})
+
+				end
+
+				if service.Workspace.StreamingEnabled == true then
+					table.insert(tab,{Text = "Streaming: [ON]"})
+				else
+					table.insert(tab,{Text = "Streaming: [OFF]"})
 				end
```

Comment:
```
				table.insert(tab,{Text = "HTTPService: ["..HTTP.CheckHttp() and "ON" or OFF.."]"})
				table.insert(tab,{Text = "Third Party Sales: ["..game.Workspace.AllowThirdPartySales and "ON" or OFF.."]"})

				table.insert(tab,{Text = "Loadstring: ["..pcall(function() loadstring("local hi = 'test'") end)  and "ON" or "OFF".."]"})

				table.insert(tab,{Text = "Streaming: ["..service.Workspace.StreamingEnabled and "ON" or OFF.."]"})
```

Reply:
Im not a LUA Expert, did you test this?

Reply:
I tested it, I found that OFF wasnt written as a string, I fixed it, and it's still giving me, Attempt to concactenate string with nil

Reply:
Oh yes I just forgot to add quotes.
It should be like "OFF"

Reply:
Also at the other place I forgot to add quotes (The first comment of this change request)

Reply:
I added them to the comments.

Reply:
Wait wait I also forgot to add it in brackets. Ill fix that.

Reply:
Fixed it. Not it should work.

Reply:
Alright, I'll test this out!

Reply:
It's still giving me the Attempt to concactenate string with nill

Reply:
Can you show me the code that puts out the error?

Reply:
Like the exact part

Reply:
The first line that displays the HTTPService, so it could be all of the lines

Reply:
Hmm... I have to test it when I can